sd_jwt/examples/complex_eidas_proposal.yml

```diff
@@ -0,0 +1,80 @@
+user_claims:
+  {
+    "verified_claims":
+      {
+        "verification":
+          {
+            "trust_framework": "eidas",
+            "assurance_level": "high",
+            "evidence": [
+                {
+                  "type": "electronic_record",
+                  "record": {
+                    "type": "eu.europa.ec.eudiw.pid.1",
+                    "source": {
+                        "organization_name": "Comune di Firenze",
+                        "organization_id": "c_d612", # italian ipa code, public service id.
+                        "country_code": "IT",
+                        "country": "Italy"
+                    },
+                    "personal_number": "9642-8fe652c8c040" # european personal id
+                  }
+                }
+            ]
+          },
+        "claims": {
+          "person_unique_identifier": "TINIT-fc0d9684-1bf0-4220-9642-8fe652c8c040",
+          "given_name": "Raffaello",
+          "family_name": "Mascetti",
+          "date_of_birth": "1922-03-13",
+          "gender": "M",
+          "place_of_birth": {
+            "country": "IT",
+            "locality": "Firenze"
+          },
+          "nationalities": [
+            "IT"
+          ]
+        }
+      },
+    "birth_middle_name": "Lello", # other optional claim not covered by eu.europa.ec.eudiw.pid.1
```

Comment:
This line just made me realized another big advantage of YAML here - we can have comments!